gradle.properties

```diff
@@ -16,7 +16,7 @@ opensaml_version=4.3.2
 jwt_version=0.12.6
 jaxb_runtime_version=4.0.5
 # TODO: we cannot update to 5.5.0 because we currently use the CP Subsystem for fenced locks, however CP Subsystem is only available to Enterprise customers
-hazelcast_version=5.4.0
+hazelcast_version=5.5.0
```

Comment:
Since we're updating to 5.5.0, can we remove this TODO?

Reply:
Thxn for noticing! Done